servers/src/main/java/tachyon/Format.java

```diff
@@ -74,14 +74,15 @@ public static void main(String[] args) throws IOException {
           Constants.RAW_TABLE_MASTER_NAME,
       };
       for (String masterServiceName : masterServiceNames) {
-        if (!formatFolder(masterServiceName + "JOURNAL_FOLDER", PathUtils.concatPath(masterJournal,
+        if (!formatFolder(masterServiceName + "_JOURNAL_FOLDER", PathUtils.concatPath(masterJournal,
```

Comment:
without this change, the folder name shown in logs are `FileSystemMasterJOURNAL_FOLDER` instead of more readable `FileSystemMaster_JOURNAL_FOLDER`.